extension/src/time_vector/pipeline.rs

```diff
@@ -255,8 +255,23 @@ pub(crate) unsafe fn pipeline_support_helper(
         UnstableTimevectorPipeline::from_datum((*old_const).constvalue, false, 0).unwrap();
     let new_pipeline = make_new_pipeline(old_pipeline, (*new_element_const).constvalue);
 
+    /// Copies a `pg_sys::Const`, since it doesn't implement `Copy` or `Clone`.
+    fn copy_const_hack(const_: &pg_sys::Const) -> pg_sys::Const {
```

Comment:
This is a workaround for an issue with `pg_sys` not having `Copy` derived for structs with `Datum`s

Reply:
This should be fixed on pgx's develop branch now.